util/bpmath/types.go

```diff
@@ -0,0 +1,19 @@
+package bpmath
+
+import (
+	"cosmossdk.io/math"
+)
+
+type Rounding uint
+
+const (
+	DOWN = iota
+	UP
+)
+
+const ONE = 10000
```

Comment:
```suggestion
const One = 10000
```
Probably don't need all caps